test/e2e/gc_e2e_test.go

```diff
@@ -3,11 +3,11 @@ package e2e
 import (
 	"context"
 	"fmt"
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
```

Comment:
nit: should be in the last block with other local imports

non-blocking.

Reply:
gotcha!